src/editor.ts

```diff
@@ -0,0 +1,216 @@
+import { IMXIcon } from "./common/icon";
+import { unpinLeft, unpinRight } from "./content";
```

Comment:
What's the point of this unused import? There's a reason why `categories` is an array. A single icon can have more than one category in its list of categories.
```suggestion
```

Reply:
I typed them by mistake and then removed them but didn't notice an import was added at the top, thanks for noticing